### ISSUE TITLE
Persist IndexedDB to disk.

### ISSUE THREAD

**Issue by ngokevin** [webmixedreality/exokit#155]
https://github.com/dumbmatter/fakeIndexedDB says only in-memory and `Functionally, it works exactly like IndexedDB except data is not persisted to disk.`

Also in Supercraft, the sites don't seem to be fetching from IndexedDB to populate the load list. Need to investigate.

**Comment by modulesio** [comment]
Yes, it was a stopgap to get the api working. localStorage should persist.

That probably means we should introduce the notion of a profile dir too.
